src/lib/api.ts

```diff
@@ -207,3 +207,13 @@ export async function changeProjectMetric(projectId: string, metric: string | nu
 		body: JSON.stringify({ metric })
 	});
 }
+
+export async function updateInstances(instances: Tables<'instances'>[]) {
```

Comment:
better function name? this just updates predictions?

Reply:
I think I'm confused by the fetch path actually

Reply:
aha yeah good catch